4.3.0 fails to cross compile on mingw

`
/go/pkg/mod/github.com/hillu/go-yara/v4@v4.3.0/rules_callback.go:165:12: could not determine kind of name for C.find_rule
cgo: 
x86_64-w64-mingw32-gcc errors for preamble:
/go/pkg/mod/github.com/hillu/go-yara/v4@v4.3.0/rules_callback.go:14:40: error: unknown type name 'uint'; did you mean 'u_int'?
   14 | static YR_RULE* find_rule(YR_RULES* r, uint rule_idx) {
      |                                        ^~~~
      |                                        u_int
`

I assume switching uint to the standard C unsigned int would fix the problem.

--- a/rules_callback.go
+++ b/rules_callback.go
@@ -11,7 +11,7 @@ package yara
 #include <yara.h>
 
 // rules_table is part of a union and as such not reachable from go code.
-static YR_RULE* find_rule(YR_RULES* r, uint rule_idx) {
+static YR_RULE* find_rule(YR_RULES* r, unsigned int rule_idx) {
 	return &r->rules_table[rule_idx];
 }
 */

